pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -340,14 +341,15 @@ func (fs *eosfs) isShareFolderChild(ctx context.Context, p string) bool {
 	return len(vals) > 1 && vals[1] != ""
 }
 
-func (fs *eosfs) GetPathByID(ctx context.Context, id *provider.ResourceId) (string, error) {
+func (fs *eosfs) GetPathByID(ctx context.Context, id *provider.Reference) (string, error) {
 	u, err := getUser(ctx)
 	if err != nil {
 		return "", errors.Wrap(err, "eos: no user in ctx")
 	}
 
 	// parts[0] = 868317, parts[1] = photos, ...
-	parts := strings.Split(id.OpaqueId, "/")
+	// FIXME @butonic REFERENCE ... umm ... 868317/photos? @ishank011 HELP?!?
+	parts := strings.Split(id.NodeId, "/")
```

Comment:
@butonic pretty sure we can change this to `fileID, err := strconv.ParseUint(id.NodeId, 10, 64)`. I can't remember any scenario where we append the path to the inode. Plus looking at the code, `GetPathByID` is never used anyway, so we should be good

Reply:
ok, but maybe not as part of this pr?